app/components/support_interface/ucas_match_action_component.rb

```diff
@@ -58,7 +58,7 @@ def button
   private
 
     def type_of_action
-      "<strong class='govuk-tag govuk-tag--yellow app-tag'>Action needed</strong> #{ACTIONS[@match.next_action][:description]}".html_safe
+      "#{govuk_tag(text: 'Action needed', colour: 'yellow')} #{ACTIONS[@match.next_action][:description]}".html_safe
```

Comment:
If you want to get rid of the string interpolation and `html_safe` you can do something like...
```
      capture do
        concat govuk_tag(text: 'Action needed', colour: 'yellow')
        concat ' '
        concat ACTIONS[@match.next_action][:description]
      end
```
Not sure that makes it any clearer. Totally optional.

Reply:
Nice! Have refactored the other content in this part of the UI, too: https://github.com/DFE-Digital/apply-for-teacher-training/pull/4428/commits/1f5598389fd6c3afeb32d065bd5fe8628af8d1be